lib/frr_zmq.c

```diff
@@ -79,13 +82,21 @@ static int frrzmq_read_msg(struct thread *t)
 
 		if (ret < 0)
 			goto out_err;
+		if (polli.revents & ZMQ_POLLERR)
```

Comment:
http://api.zeromq.org/4-2:zmq-poll

This documentation specifically states that ZMQ_POLLERR -> 'For ØMQ sockets this flag has no effect if set in events, and shall never be returned in revents by zmq_poll().'

Are you actually seeing this returned?